### PR TITLE
CIF-2937 - 2022.08.0 Release Post Actions

### PR DESCRIPTION
**Pull request by LSantha**
Update CIF components to the latest release.

## Related Issue

CIF-2937

## How Has This Been Tested?

Manually.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/pom.xml [hunk 1]
```diff
@@ -76,8 +76,8 @@
 #end
         <core.wcm.components.version>2.19.0</core.wcm.components.version>
 #if ( $includeCommerce == "y" )
-        <core.cif.components.version>2.10.0</core.cif.components.version>
-        <magento.graphql.version>9.0.0-magento242ee</magento.graphql.version>
+        <core.cif.components.version>2.11.0</core.cif.components.version>
```
Comment (buuhuu) [inline src/main/archetype/pom.xml:79]: You should probably update the sdk version as well as this is not a hard dependency.

Also I am not sure if the graphql.client dependency is event required anymore (considering the SDK is required for cloud and onprem now)

Comment (LSantha) [reply src/main/archetype/pom.xml:79]: The SDK version was already updated here https://github.com/adobe/aem-project-archetype/commit/a61384f76eaeed7b41a98264f61d0c86e84f7061#diff-8d2547db69a77df225119beca89eeb18180b73f993ae465fc00d5e5b32985486R85 .

The graphql client can be removed indeed and dependencies can be simplified.
I've updated the PR.

Comment (buuhuu) [reply src/main/archetype/pom.xml:79]: what about the all package? The graphql-client is still embedded there. This should not be necessary anymore either.

Comment (LSantha) [reply src/main/archetype/pom.xml:79]: Done.

**Review by buuhuu — APPROVED** [review]

**Pull request merged**